### DISCUSSION TITLE
Transfer license from Firefox to Chrome?

### DISCUSSION THREAD

**Discussion by jamesd1** [aclap-dev/video-downloadhelper#917] [License & Purchase requests]
### What's your issue?

I changed my browser from Firefox to Chrome.  Can I transfer my Firefox license to Chrome browser?

mystic1mus 

_No response_

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry it's not possible to swap licenses.
